packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -136,6 +136,8 @@ export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'con
   id?: string;
   /** CSS fade transition animation duration */
   animationDuration?: number;
+  /** Determines whether the tooltip is hidden regardless of visibility **/
+  hidden?: boolean;
```

Comment:
I'm not sure whether we should introduce this prop, mainly because we already have an "isVisible" prop for manual control of a Tooltip.

Rather than needing to create this prop, we could just conditionally return either the  "full" Truncate content (if there is no truncation; basically everything that is currently being passed to Tooltip) or the "full" Truncate content wrapped in the Tooltip (if there is truncation).

We do something similar in ModalBoxTitle.